docs/connector-development/tutorials/adding-incremental-sync.md

```diff
@@ -37,6 +37,39 @@ def discover():
     print(json.dumps(airbyte_message))
 ```
 
+Also, create a file called `incremental_configured_catalog.json` with the following content:
+```javascript
+{"type": "RECORD", "record": {"stream": "stock_prices", "data": {"date": "2023-01-25", "stock_ticker": "TSLA", "price": 144.43}, "emitted_at": 1674812095000}}
```

Comment:
Is the record needed here, or just the streams below?

Reply:
Good catch!
It was a copy-paste error.
Removed the `RECORD` message 😊